use_map_topic             : true
first_map_only            : false
gui_publish_rate          : 10

laser_min_range           : 0.15
laser_max_range           : 8.0
laser_max_beams           : 30

min_particles             : 70
max_particles             : 5000

kld_err                   : 0.05
kld_z                     : 0.99

laser_model               : LASER_MODEL_LIKELIHOOD_FIELD_PROB
z_hit                     : 0.5
z_rand                    : 0.5
sigma_hit                 : 0.2
lambda_short              : 0.1
laser_likelihood_max_dist : 5.0
do_beamskip               : false
beam_skip_distance        : 0.5
beam_skip_threshold       : 0.3
beam_skip_error_threshold : 0.9


odom_model   : ODOM_MODEL_OMNI
odom_alpha1  : 0.005
odom_alpha2  : 0.005
odom_alpha3  : 0.01
odom_alpha4  : 0.005
odom_alpha5  : 0.003

update_min_d      : 0.05
update_min_a      : 0.03

resample_interval   : 1
transform_tolerance : 0.1
recovery_alpha_slow : 0.001
recovery_alpha_fast : 0.1

use_global_localization : false
random_heading : true
laser_filter_weight : 0.4

max_uwb_particles : 10
uwb_cov_x : 0.06
uwb_cov_y : 0.06
resample_uwb_factor : 4.0
